service/mcs/persistent/src/main/resources/create_cassandra_schema.cql

```diff
@@ -41,4 +41,11 @@ CREATE TABLE representation_versions (
 	creation_date timestamp,
 	PRIMARY KEY (cloud_id, schema_id, version_id)
 );
-CREATE INDEX representations_provider_id ON representation_versions (provider_id);
\ No newline at end of file
+CREATE INDEX representations_provider_id ON representation_versions (provider_id);
+
+CREATE TABLE data_sets_representations_names(

```

Comment:
Why is this done like that?? It's not as we discussed in the document...
And with the naming you don't need to put plural in all the words, "data_set_representation_names" and "representation_names". And "dataset" is accepted in English without space or underscore in between, like "database".